inclusive-design/wecount.inclusivedesign.ca
# feat: migrate News and Views to Netlify CMS

## Description

Completes migration to Netlify CMS, adding Eleventy and CMS collections for News and Views.

Reformats table view found [here](https://wecount.inclusivedesign.ca/views/environmental-scan-assessing-inclusionary-practice-in-canadian-data-services/) into list as discussed Monday.

## Steps to test

1. Run locally (`npm start`).
2. Test CMS interactions for News and Views.
2. Test News and Views pages, individual items, search and filtering.

**Expected behavior:** All behaviour is consistent with existing site, all content can be edited and previewed.

## Additional information

Not applicable.

## Related issues

Not applicable.
